.cargo/config

```diff
@@ -3,3 +3,9 @@
 
 [alias]
 xtask = "run --package xtask --"
+
+[target.xtensa-esp32s3-none-elf]
+rustflags = [
+    "-C", "force-frame-pointers",
+    "-C", "target-feature=-fp",
+]
```

Comment:
Do we still need this in the mcu-board-support file as well then? Might be worth a comment to explain why we set these flags.

Reply:
Yes, because of 
 - `build-std = ["core", "alloc"]`
 - `-Clink-arg=-Wl,-Tlinkall.x`

Which we need to pass to the rust build, but not the C++ build because that's done differently.

Reply:
Fair point, seems indeed that this needs to be separate. I still think a comment would make sense here (that this is to work around a toolchain bug).